src/java/io/bazel/rulesscala/test_discovery/BUILD

```diff
@@ -7,5 +7,5 @@ scala_library(
         "FilteredRunnerBuilder.scala",
     ],
     visibility = ["//visibility:public"],
-    deps = ["//external:io_bazel_rules_scala/dependency/junit/junit"],
+    deps = ["@io_bazel_rules_scala//testing/toolchain:junit_classpath"],
```

Comment:
This isn't a problem because the declare deps provider uses the "bind" notation deps?

My guess is that we'll drop the bonds later on after users introduce their own toolchains, right?

If so then won't we have a problem then? a user configures their own toolchain but this is hardwired to our toolchain. Feel free to correct my understanding if I'm simply wrong

Reply:
This target depends on the toolchain type, so standard toolchain resolution will work